pandas/tests/frame/methods/test_drop_duplicates.py

```diff
@@ -441,7 +441,6 @@ def test_drop_duplicates_null_in_object_column(nulls_fixture):
     tm.assert_frame_equal(result, df)
 
 
-@pytest.mark.parametrize("keep", ["first", "last", False])
 def test_drop_duplicates_series_vs_dataframe(keep):
```

Comment:
cause this is the same as the fixture? 🤔 might be an idea for an automated test

Reply:
Yup same as this one https://github.com/pandas-dev/pandas/blob/cca262d35b881d327e26782da699c1fe405c1c24/pandas/conftest.py#L304-L310

Yeah it would be great to have a check to answer "Does a @pytest.mark.parametrize return the same params as an existing fixture"